concepticondata/conceptlists/Tao-2023-90.tsv

```diff
@@ -0,0 +1,91 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Tao-2023-90-1	1	ashes	646	ASH
+Tao-2023-90-2	2	belly	1251	BELLY
+Tao-2023-90-3	3	big	1202	BIG
+Tao-2023-90-4	4	bird	937	BIRD
+Tao-2023-90-5	5	bite	1403	BITE
+Tao-2023-90-6	6	black	163	BLACK
+Tao-2023-90-7	7	blood	946	BLOOD
+Tao-2023-90-8	8	bone	1394	BONE
+Tao-2023-90-9	9	breast	1402	BREAST
+Tao-2023-90-10	10	burn tr.	2102	BURN
+Tao-2023-90-11	11	claw(nail)	72	CLAW
+Tao-2023-90-12	12	cloud	1489	CLOUD
+Tao-2023-90-13	13	cold	1287	COLD
+Tao-2023-90-14	14	come	1446	COME
+Tao-2023-90-15	15	die	1494	DIE
+Tao-2023-90-16	16	dog	2009	DOG
+Tao-2023-90-17	17	dry	1398	DRY
+Tao-2023-90-18	18	ear	1247	EAR
+Tao-2023-90-19	19	earth	1228	EARTH (SOIL)
+Tao-2023-90-20	20	eat	1336	EAT
+Tao-2023-90-21	21	egg	744	EGG
+Tao-2023-90-22	22	eye	1248	EYE
+Tao-2023-90-23	23	fat n.	323	FAT (ORGANIC SUBSTANCE)
+Tao-2023-90-24	24	feather	1201	FEATHER
+Tao-2023-90-25	25	fire	221	FIRE
+Tao-2023-90-26	26	fish	227	FISH
+Tao-2023-90-27	27	fly v.	1441	FLY (MOVE THROUGH AIR)
+Tao-2023-90-28	28	foot	1301	FOOT
+Tao-2023-90-29	29	full	1429	FULL
+Tao-2023-90-30	30	give	1447	GIVE
+Tao-2023-90-31	31	good	1035	GOOD
+Tao-2023-90-32	32	green	1425	GREEN
+Tao-2023-90-33	33	hair	1040	HAIR
+Tao-2023-90-34	34	hand	1277	HAND
+Tao-2023-90-35	35	head	1256	HEAD
+Tao-2023-90-36	36	hear	1408	HEAR
+Tao-2023-90-37	37	heart	1223	HEART
+Tao-2023-90-38	38	horn	1393	HORN (ANATOMY)
+Tao-2023-90-39	39	I	1209	I
+Tao-2023-90-40	40	kill	1417	KILL
+Tao-2023-90-41	41	knee	1371	KNEE
+Tao-2023-90-42	42	know	1410	KNOW (SOMETHING)
+Tao-2023-90-43	43	leaf	628	LEAF
+Tao-2023-90-44	44	liver	1224	LIVER
+Tao-2023-90-45	45	long	1203	LONG
+Tao-2023-90-46	46	louse	1392	LOUSE
+Tao-2023-90-47	47	man	1554	MAN
+Tao-2023-90-48	48	many	1198	MANY
+Tao-2023-90-49	49	meat	2615	FLESH OR MEAT
+Tao-2023-90-50	50	moon	1313	MOON
+Tao-2023-90-51	51	mountain	639	MOUNTAIN
+Tao-2023-90-52	52	mouth	674	MOUTH
+Tao-2023-90-53	53	name	1405	NAME
+Tao-2023-90-54	54	neck	1333	NECK
+Tao-2023-90-55	55	new	1231	NEW
+Tao-2023-90-56	56	night	1233	NIGHT
+Tao-2023-90-57	57	nose	1221	NOSE
+Tao-2023-90-58	58	not	1240	NOT
+Tao-2023-90-59	59	one	1493	ONE
+Tao-2023-90-60	60	person	683	PERSON
+Tao-2023-90-61	61	rain	2108	RAINING OR RAIN
+Tao-2023-90-62	62	red	156	RED
+Tao-2023-90-63	63	road	667	ROAD
+Tao-2023-90-64	64	root	670	ROOT
+Tao-2023-90-65	65	round	1395	ROUND
+Tao-2023-90-66	66	see	1409	SEE
+Tao-2023-90-67	67	seed	714	SEED
+Tao-2023-90-68	68	sit	1416	SIT
+Tao-2023-90-69	69	skin	763	SKIN
+Tao-2023-90-70	70	sleep	1585	SLEEP
+Tao-2023-90-71	71	small	1246	SMALL
+Tao-2023-90-72	72	smoke	778	SMOKE (EXHAUST)
```

Comment:
Do we have any information regarding the grammatical category of the gloss?

Reply:
Based on the Swadesh list, this mapping is correct.